### DISCUSSION TITLE
Error encountered in VS2022 installation

### DISCUSSION THREAD

**Discussion by chengwan0203** [community/community#36296] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/42882190/196023837-6f8c3b9d-dcfc-41ca-8983-1a33ae518228.png)

hello，

I try to install copilot in VS2022, but keep getting an error

Is there any solution please?

**Comment by iamcjwilliams** [comment]
@chengwan0203 Here are steps for login from a clean state:

1. Close Visual Studio 2022.
2. Delete C:\Users<insert username>\AppData\Local\github-copilot\hosts.json
3. Log out of GitHub in the web browser.
4. Open Visual Studio.

**Comment by ossanna16** [comment]
Hey there @chengwan0203 and thank you for asking a good question! I'm glad you found our community 🙂
